### ISSUE TITLE
Automatic re-focus breaks a11y

### ISSUE THREAD

**Issue by marksreeves** [BorderTech/wcomponents#501]
A number of components call setFocussed() to automatically re-focus themselves when a view is rendered. For example a WButton will regain focus if it is in the view which is loaded after it is used to invoke a https POST "round trip". This was originally introduced at the behest of an accessibility consultant who appears to have not been aware of the meaning of SC 2.4.3

This **must** be removed as it is by no means guaranteed that the control should retain focus. Under **most** circumstances only one of the following two scenarios are valid after a "round trip":
1. the same view is displayed with validation errors, in which case the validation error list should have default focus; or
2. a new view is displayed, in which case focus **must not** be set to the old button.
## Example

A multi-view application has a WButton used to save each view in a sequential process. Each step of the process is a unique view but the "Save" button is shared. It would be completely inappropriate for this button to retain focus after causing the sequence to proceed from step `N` to step `N + 1`.
## Proposed solution

Remove all in-built calls to `setFocussed()` but retain `setFocussed()` so that each use-case may be ascertained on its merits thereby moving the determination of appropriate focus retention to the only place it can be adequately specified: the consuming application.

**Comment by marksreeves** [comment]
Re-labelled as a bug as this does cause an accessibility issue under common use-cases.
